Proofread update/upgrade/migrate chapter (#2669)
centreon/centreon-documentation: versioned_docs/version-23.10/migrate/poller-display-to-remote-server.md

If the module **centreon-poller-display-central-1.6.x** is installed:

> 1. Go to the **Administration > Extensions > Modules** menu and uninstall 

Reviewer: ```suggestion
1. Go to **Administration > Extensions > Modules** and uninstall 
```

## Migrate a Poller from Poller Display module to Remote Server

> 1. Go to the **Administration > Extensions > Modules** menu and uninstall

Reviewer: ```suggestion
1. Go to **Administration > Extensions > Modules** and uninstall the
```

Refer to the [migration procedure](../migrate/migrate-from-3-4.md).

    > If you use Centreon EMS modules, you must update the repositories. Be sure to

>     > contact Centreon support and request new licenses.

Reviewer: ```suggestion
    > contact your Centreon support team and request new licenses.
```

2. Select the IP address if multiple network interfaces exist:

    ```shell
    Which IP do you want to use as CURRENT NODE IP?
    1) 192.168.0.2
    2) 192.168.0.3

>     1.

Reviewer: ```suggestion
    1
```

    ```

3. Then validate the information:

    ``` shell

>     Summary of the information that will be sent:

Reviewer: ```suggestion
    Summary of the informations that will be send:
```

>     API connection:

Reviewer: ```suggestion
    Api connection:
```

    type: remote
    address: 192.168.0.2
    

>     Do you want to register this server with that information? (y/n)y

Reviewer: ```suggestion
    Do you want to register this server with those informations? (y/n)y
```

    ```

4. Add additional information to enable future communication between your Remote Server and its Central.

> Please provide the required information to convert your platform to Remote:

Reviewer: ```suggestion
Provide the required information to convert your platform to a remote server:
```

2020-10-16T17:19:37+02:00 [INFO]: The CURRENT NODE 'remote: 'remote-1@192.168.0.2' has been converted and registered successfully.
```

### Main error messages

``` shell

> 2020-10-20T10:11:15 PM+02:00 [ERROR]: Invalid credentials

Reviewer: ```suggestion
2020-10-20T10:23:15+02:00 [ERROR]: Invalid credentials
```
